app/src/main/java/edu/drexel/lapcounter/lapcounter/backend/lapcounter/LapCounter.java

```diff
@@ -0,0 +1,105 @@
+package edu.drexel.lapcounter.lapcounter.backend.lapcounter;
```

Comment:
How did this file get here? I see no merge commit so it looks like you just copied and pasted in many files